### ISSUE TITLE
Support for JSON

### ISSUE THREAD

**Issue by glynd** [awslabs/aws-sam-local#36]
Doesn't appear to support JSON yet....
```
aws-sam-local validate -t sam_build.json 
2017/08/14 11:12:07 PANIC: YAML contains malformed structures
panic: PANIC: YAML contains malformed structures
```

**Comment by pesama** [comment]
Hi Glyn,

Working on it!

**Comment by PaulMaddox** [comment]
We're rewriting some of the library we use for CloudFormation/SAM parsing (github.com/awslabs/goformation). Part of this rewrite will include JSON support.

The dev work is mostly done, in the 0.1.0 branch of goformation, however we've got a bit more work to do before we can start using it in SAM Local. Shouldn't be much longer.

**Comment by sanathkr** [comment]
Fixed in [v0.2.0](https://github.com/awslabs/aws-sam-local/releases/tag/v0.2.0)
